packages/neuron-ui/src/styles/layout.scss

```diff
@@ -1,21 +1,22 @@
 body {
   display: grid;
-  grid-template:
-    'header header header'40px 'sidebar mainContainer mainContainer'1fr 'sidebar mainContainer mainContainer'1fr 'sidebar mainContainer mainContainer'1fr / 240px 1fr 1fr;
+  grid-template: 'sidebar header' 60px 'sidebar content' 1fr 'sidebar content' 1fr / 240px 1fr;
+  column-gap: 40px;
 
   height: 100vh;
 }
 
-header {
-  grid-area: header;
+.sidebar {
+  grid-area: sidebar;
 }
 
-aside {
-  grid-area: sidebar;
+.header {
+  grid-area: header;
 }
 
-.mainContainer {
-  grid-area: mainContainer;
+.main-content {
+  grid-area: content;
+  overflow: auto;
 }
 
 .notification {
```

Comment:
If `body.grid-template` is set to ` 'sidebar header' 60px 'sidebar content' 1fr 'sidebar content' 1fr / 240px 1fr;`, `.notification`'s `grid-area` should be modified to proper position.

Reply:
Makes sense and we can adjust that later.

I'm think perhaps we should place `notification` inside header to make it more easier to be noticed.

Reply:
Adjust it later.

I added it as a container for now, not necessary. I prefer using electron api for notification.